forge-gui/res/editions/Warhammer 40,000 Commander.txt

```diff
@@ -331,21 +331,23 @@ ScryfallCode=40K
 [tokens]
 1 w_2_2_astartes_warrior_vigilance @Dmitriy Mironov
 2 w_1_1_soldier @Lixin Yin
-2★ w_1_1_soldier @Lixin Yin
 3 w_1_1_soldier @Lixin Yin
-3★ w_1_1_soldier @Lixin Yin
 4 w_1_1_soldier @Lixin Yin
-4★ w_1_1_soldier @Lixin Yin
+# Not implemented (Squad tokens)
+# 5 space_marine_devastator @Dmitriy Mironov
+# 6 ultramarines_honour_guard @Jake Murray
+# 7 zephyrim @Jon Neimeister
+# 8 sicarian infiltrator @Xavier Ribeiro
 9 u_1_1_tyranid_gargoyle_flying @David Astruga
+# Not implemented (Squad tokens)
+# 10 vanguard_suppressor @Fajareka Setiawan
+# 11 arco-flagellant @Xavier Ribeiro
```

Comment:
try to use `[other]` section for new Squad image keys

Reply:
Doesn't work in either tokens or other.